tools/deployment-cli-tools/ch_cli_tools/dockercompose.py

```diff
@@ -0,0 +1,320 @@
+            self.inject_entry_points_commands(values, image_path, app_path)
+
+        if len(image_paths) > 1:
+            logging.warning('Multiple Dockerfiles found in application %s. Picking the first one: %s', app_name,
+                            image_paths[0])
+        if KEY_HARNESS in values and 'dependencies' in values[KEY_HARNESS] and 'build' in values[KEY_HARNESS]['dependencies']:
+            build_dependencies = values[KEY_HARNESS]['dependencies']['build']
+        else:
+            build_dependencies = []
+
+        if len(image_paths) > 0:
+            image_name = image_name_from_dockerfile_path(os.path.relpath(
+                image_paths[0], os.path.dirname(app_path)), base_image_name)
+
+            values['image'] = self.image_tag(
+                image_name, build_context_path=app_path, dependencies=build_dependencies)
+        elif KEY_HARNESS in values and not values[KEY_HARNESS].get(KEY_DEPLOYMENT, {}).get('image', None) and values[
+                KEY_HARNESS].get(KEY_DEPLOYMENT, {}).get('auto', False):
+            raise Exception(f"At least one Dockerfile must be specified on application {app_name}. "
+                            f"Specify harness.deployment.image value if you intend to use a prebuilt image.")
+
+        task_images_paths = [path for path in find_dockerfiles_paths(
+            app_path) if 'tasks/' in path]
+        values[KEY_TASK_IMAGES] = values.get(KEY_TASK_IMAGES, {})
+
+        if build_dependencies:
+            for build_dependency in values[KEY_HARNESS]['dependencies']['build']:
+                if build_dependency in self.base_images:
+                    values[KEY_TASK_IMAGES][build_dependency] = self.base_images[build_dependency]
+
+        for task_path in task_images_paths:
+            task_name = app_name_from_path(os.path.relpath(
+                task_path, app_path.parent))
+            img_name = image_name_from_dockerfile_path(task_name, base_image_name)
+
+            # values[KEY_TASK_IMAGES][task_name] = self.image_tag(
+            #     img_name, build_context_path=task_path, dependencies=values[KEY_TASK_IMAGES].keys())
+            # values.setdefault(KEY_TASK_IMAGES_BUILD, {})[task_name] = {
+            #     'context': os.path.relpath(task_path, self.dest_deployment_path.parent),
+            #     'dockerfile': 'Dockerfile',
+            # }
+
+            # values[KEY_TASK_IMAGES][task_name] = {
+            #     'name': self.image_tag(img_name, build_context_path=task_path, dependencies=values[KEY_TASK_IMAGES].keys()),
+            #     # 'context': os.path.relpath(task_path, self.dest_deployment_path.parent),
+            #     # 'dockerfile': 'Dockerfile',
+            # }
+
+            values[KEY_TASK_IMAGES][task_name] = self.image_tag(img_name, build_context_path=task_path, dependencies=values[KEY_TASK_IMAGES].keys())
+
+        return values
+
+    def inject_entry_points_commands(self, helm_values, image_path, app_path):
```

Comment:
I'm not sure this is necessary: we do command overriding on skaffold to enable debugging that doesn't work well with unicorn